nilearn/datasets/struct.py

```diff
@@ -106,38 +114,177 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
-def load_mni152_template():
-    """Load skullstripped 2mm version of the MNI152 originally distributed
-    with FSL.
+def load_mni152_template(resolution=2):
+    """This function takes the skullstripped, re-scaled 1mm-resolution version
```

Comment:
not important but for next time [pep 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) asks for the summary line of docstrings to really fit on one line; it can be a good idea to respect it in case the doc rendering tool comes to rely  on it being respected at some point

Reply:
OK. I added a one summary line at the beginning of the description of these functions in their corresponding docstr.